packages/core/src/testing/webhook.ts

```
@@ -0,0 +1,343 @@
+        body
+      };
+      
+    case 'malformed_payload':
+      return {
+        headers: createMockWebhookHeaders('invalid json', secret),
+        body: 'invalid json'
+      };
+      
+    case 'expired_timestamp':
+      // Create a timestamp from 1 hour ago (older than the 5-minute tolerance)
+      const expiredTimestamp = new Date(Date.now() - 60 * 60 * 1000);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)